aws-single-instance-resiliency/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: nxrm-aws-resiliency
```

Comment:
I would have expected the name of this directory to match the name here in the file.

Reply:
Do the directory names have to match the chart names in order for it to work?

Reply:
I don't know if the need to match. i've not tried to mismatch them. you could probably prove that it'll work fine, BUT I feel it might be easier for us to navigate to easily stumble into the 'charts/nxrm-aws-resiliency' directory to find the chart we'd `helm install sonatype/nxrm-aws-resiliency`. The name feels hidden in the Charts.yaml as it is now.

Reply:
I see your point. That would mean both directories in [here](https://github.com/sonatype/nxrm3-helm-repository) should be renamed to 'nexus-repository-manager' (for existing chart) and 'nxrm-aws-resiliency' for the new AWS one.

Reply:
Renamed: https://github.com/sonatype/nxrm3-helm-repository/pull/9/commits/2aa0f5dbf98e0757c61bda586bf64dbe6a0c570b